components/collection/drop/GenerativePreview.vue

```diff
@@ -136,31 +137,29 @@ const getHash = (isDefault?: boolean) => {
   return blake2AsHex(initialValue, 256, null, true)
 }
 
-const generativeImageUrl = ref(
-  accountId.value ? `${props.drop.content}/?hash=${getHash(true)}` : '',
-)
+const generativeImageUrl = ref('')
 
 const isLoading = ref(false)
 
 const displayUrl = computed(() => {
   return generativeImageUrl.value || props.drop.image
 })
-const generateNft = (isDefault: boolean = false) => {
+const generateNft = () => {
   isLoading.value = true
-  const metadata = `${props.drop.content}/?hash=${getHash(isDefault)}`
+  const metadata = `${props.drop.content}/?hash=${getHash()}`
   generativeImageUrl.value = metadata
-  emit('generation:start', { image: generativeImageUrl.value, isDefault })
+  emit('generation:start', { image: generativeImageUrl.value })
 
   setTimeout(() => {
     isLoading.value = false
-    emit('generation:end', isDefault)
+    emit('generation:end')
   }, 3000)
```

Comment:
What if someone clicks sooner?

Reply:
The button would be disabled during the 3 seconds, so anyone could not click sooner in this case.

Reply:
so it is response from iframe + 3 seconds?

Reply:
It's responses after clicking the button plus 3 seconds.

Reply:
Now it is response from iframe loaded. Updated at commit https://github.com/kodadot/nft-gallery/pull/9516/commits/7167752f670657ae6bd0a92ccafd179a9c74416